sdk/textanalytics/azure-ai-textanalytics/tests/testcase.py

```diff
@@ -128,7 +128,7 @@ def create_text_analytics_client(self, **kwargs):
 def text_analytics_account():
     test_case = AzureTestCase("__init__")
     rg_preparer = ResourceGroupPreparer(random_name_enabled=True, name_prefix='pycog')
-    text_analytics_preparer = CognitiveServicesAccountPreparer(random_name_enabled=True, name_prefix='pycog')
+    text_analytics_preparer = CognitiveServicesAccountPreparer(random_name_enabled=True, name_prefix='pycog', location='CentralUSEUAP')
```

Comment:
should we change this to westus2? or is there no difference between the canary region and westus2 now?

Reply:
I just did a run with westus2 in the hopes of changing it, I saw some flakiness (see [this](https://teams.microsoft.com/l/message/19:d6ff4003f5c848a2a2d6dcdfc0ebd497@thread.skype/1590513491314?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1590513173426&teamName=Azure%20SDK&channelName=Service%20-%20Text%20Analytics&createdTime=1590513491314) teams link). I can still update this here soon though

Reply:
updated